src/main/java/io/unlogged/core/bytecode/method/JSRInliner.java

```diff
@@ -20,10 +20,10 @@ public class JSRInliner extends JSRInlinerAdapter {
 	 * @param signature is a generics signature
 	 * @param exceptions specifies exceptions thrown by the method 
 	 */
-	public JSRInliner(MethodTransformer mv, int access, String name, String desc, String signature, String[] exceptions) {
+	public JSRInliner(MethodTransformer transfromer_probed, int access, String name, String desc, String signature, String[] exceptions) {
 		// The second parameter is null so that the object delays the execution of the given MethodTransformer
 		super(Opcodes.ASM5, null, access, name, desc, signature, exceptions);
-		this.analysis = mv;
+		this.analysis_probed = transfromer_probed;
```

Comment:
camelCase names for parameters and fieldNames

Reply:
also resolved.